src/lib/isVAT.js

```diff
@@ -0,0 +1,25 @@
+import assertString from './util/assertString';
+
+export const vatMatchers = {
+  'en-GB': [
+    /^GB\d{3} \d{4} ([0-8][0-9]|9[0-6])$/,
+    /^GB\d{9} \d{3}$/,
+    /^GBGD[0-4][0-9]{2}$/,
+    /^GBHA[5-9][0-9]{2}$/,

```

Comment:
Possible to make this one-line regex instead, to save on the matching loop later?

Reply:
Yep that works

Reply:
For the country codes, I'm cool going for [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) which would be `GB` for the UK.